py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -351,6 +388,15 @@ def to_trt_weights(
         )
 
 
+def record_weight_in_ctx(
```

Comment:
This could either be a method of the ConversionCtx or it can just be done in to_trt_weights